core/templates/components/shared-component.module.ts

```diff
@@ -83,6 +86,7 @@ import { ProfileLinkTextComponent } from
     LazyLoadingComponent, LoadingDotsComponent, SocialButtonsComponent,
     ProfileLinkImageComponent, ProfileLinkTextComponent,
     // These elements will remain here even after migration.
+    TakeBreakModalComponent,
```

Comment:
add the `TakeBreakModalComponent` in the exports as well I think.

Reply:
I already tried that but still wasn't working.